programs/Autolevels_ReadMe.txt

```diff
@@ -1,230 +0,0 @@
-Autolevels 0.6 for Avisynth, Jan. 2011

```

Comment:
We need to keep it somewhere.

Reply:
I can add it back. There are however a lot of readme files and licenses that could be added though - the plugins each have their own readme, and some plugins (like `assrender`) have an entire folder of licenses that go with it. Should we add all plugin readmes/licenses?

Reply:
I've added a docs folder with all licenses and readmes from programs and plugins that we use. We can roll back and take stuff out if we don't want everything.

Reply:
Oh, right! Would it be too annoying to put them in folders corresponding to what they belong to?

Reply:
Done!